README.md

```diff
@@ -5,6 +5,19 @@ Junction
 
 Junction is a software to manage proposals, reviews, schedule, feedback during conference.
 
+Project Setup using Docker
+--------------------------
+
+Prerequisites:
+1. Docker: You can download and install Docker from the official website at https://www.docker.com/get-started.
+
+Instructions:
+1. Copy the .env.sample file to a new .env file by running the following command: ```cp .env.sample .env```
+2. Create a local development settings file by running the following command: ```cp settings/dev.py.sample settings/dev.py```
+3. Build the junction_local image using the following command: ```docker build -t junction_local .```
+4. Start the project by running the following command: ```docker-compose up```
+5. Access the application at https://localhost:8888.
+
```

Comment:
I think we need to add a note for the apple silicon machines as suggested below.
```suggestion
>Note:  
> For Apple Silicon machines set the following environment variable before building the junction image.
> ```sh
> export DOCKER_DEFAULT_PLATFORM=linux/amd64
> ```

```

Reply:
Thanks @leshravnya for the suggestion.
The  [issue](https://github.com/psycopg/psycopg2/issues/1360) exist in psycopg 2.9.5 which got resolved in version 2.9.6, So I have upgraded the psycopg version to 2.9.6.

Reply:
Oh that's great then. Let me try it out.